app/models/card.rb

```diff
@@ -4,12 +4,30 @@ class Card < ActiveRecord::Base
 
   before_validation :set_default_review_date, :strip_whitespaces, on: :create
 
+  scope :random, -> { order("RANDOM()") }
+  scope :need_to_review, -> { where("review_date <= ?", Time.zone.now )
```

Comment:
Block body expression is on the same line as the block start.<br>Space inside parentheses detected.